src/object/obj_internal.h

```diff
@@ -592,7 +593,8 @@ obj_retry_error(int err)
 	       err == -DER_EXCLUDED || err == -DER_CSUM ||
 	       err == -DER_TX_BUSY || err == -DER_TX_UNCERTAIN ||
 	       err == -DER_NEED_TX || err == -DER_NOTLEADER ||
-	       err == -DER_UPDATE_AGAIN || daos_crt_network_error(err);
+	       err == -DER_UPDATE_AGAIN || err == -DER_NVME_IO ||
```

Comment:
I guess this is rare but it makes my eyes hurt.  Seems like we could either change these error codes (so they are all in the same range) or we could define a function to convert the error code into an index to a static array that gives us true or false.  This would take far fewer checks with our FOREACH macro setup.  Anyway, not a comment on this patch, just something I don't like seeing.

So it would be something like
idx = d_err2idx(errno);
return error_is_retryable[idx];

We already do something similar for d_errstr conversions.

Reply:
agreed, let's fix it in a separate patch.